app/views/shelters/show.html.erb

```diff
@@ -8,4 +8,16 @@
     <p><%= link_to "Edit #{@shelter.name}", "/shelters/#{@shelter.id}/edit" %></p>
     <p><%= link_to "Delete #{@shelter.name}", "/shelters/#{@shelter.id}", method: :delete %></p>
   </section>
+
+  <h2>Reviews for <%= @shelter.name %></h2>
+  <% @shelter.reviews.each do |review| %>
+    <section id="review-<%= review.id %>">
+      <%= review.title %>
+      <%= review.rating %>
+      <%= review.content %>
+      <% if review.image %>
```

Comment:
Once again one of those things I didn't know you could do. Excellent use of the if statement in the embedded ruby.